mysql-test/suite/mariabackup/backup_ssl.test

```diff
@@ -14,13 +14,6 @@ exec $XTRABACKUP  --prepare --target-dir=$targetdir;
 DROP USER backup_user;
 rmdir $targetdir;
 
-echo #;
-echo # MDEV-31855 validate ssl certificates using client password in the internal client;
-echo #;
-# fails to connect, passwordless root
-echo # tcp ssl ssl-verify-server-cert;
-error 1;
-exec $XTRABACKUP --no-defaults --protocol=tcp --user=root --port=$MASTER_MYPORT --backup --target-dir=$targetdir;
```

Comment:
why did you move it? you could've added host=127.0.0.2 as you did elsewhere and keep the test here